CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Version v1.0.0
+
+### Changes
+
+  * [S1T-1321](https://jira-sage.valiantyscloud.net/browse/S1T-1321) - Add methods for string to binary conversion (and vice versa)
```

Comment:
Remove reference to jira this gem is open source. May be useful add prior version info from releases

Reply:
Done.